### PR TITLE
Files addition with values filename and patch.  File type addition added. 

### PR DESCRIPTION
**Pull request by Courtneyp123**

### CODE REVIEW

File: src/main/java/com/capitalone/dashboard/webhook/github/GitHubCommitV3.java [hunk 1]
```diff
@@ -221,6 +222,12 @@ public String process(JSONObject jsonObject) throws MalformedURLException, Hygie
                 commit.setFilesModified((List) cObj.get("modified"));
             }
 
+            List<RepoFile> files = (List<RepoFile>) cObj.get("files");
+            for (RepoFile file : files) {
+                String fileType = file.getFilename().split("[.]")[1];
```
Comment (rvema) [inline src/main/java/com/capitalone/dashboard/webhook/github/GitHubCommitV3.java:227]: the variable is initialized inside a loop which is useless. Can you check this?

**Review by rvema — CHANGES_REQUESTED** [review]
please address the review comments

File: src/main/java/com/capitalone/dashboard/webhook/github/GitHubCommitV3.java (hunk 1 above)
Comment (Courtneyp123) [reply src/main/java/com/capitalone/dashboard/webhook/github/GitHubCommitV3.java]: Removing for now as Daniel is using the file type in his story.

File: src/main/java/com/capitalone/dashboard/webhook/github/GitHubCommitV3.java [hunk 2]
```diff
@@ -221,6 +222,13 @@ public String process(JSONObject jsonObject) throws MalformedURLException, Hygie
                 commit.setFilesModified((List) cObj.get("modified"));
             }
 
+            if (cObj.get("pullNumber") instanceof String) {
```
Comment (nireeshT) [inline src/main/java/com/capitalone/dashboard/webhook/github/GitHubCommitV3.java:225]: remove set pullNumber

**Pull request closed without merging**